# Example: saved capture results on GitHub

This notebook was **run on Almera** against the shared Qwen (`:8270`), then **saved and pushed**.
GitHub does not re-run the model. It only displays the output that was stored in this file.

- **Note:** Patient received FOLFOX. Disease progressed. FOLFOX was discontinued.
- **Layer:** 20 (block index, not “L2”)
- **L2 in the printout:** Euclidean norm of the last-token residual

Teaching notebook: [`../notebooks/02_observe/representation/01_capture.ipynb`](../notebooks/02_observe/representation/01_capture.ipynb)


In [1]:
look.cmd_capture(model, tok, LAYER, NOTE)


prompt tokens: 15
shape last-token: (1, 15, 3584)  [printing last token only]
layer 20 last-token L2: 114.6233
first 10: [0.75879, -2.01758, 0.82471, 0.63623, -0.3064, -0.81738, 1.39551, -0.2207, -2.35547, -0.08154]
(used Qwen already loaded on :8270 pid=780195 — no second load)


## Claim boundary

- We **can** say: at L20, this note’s last-token residual has dim 3584 and L2 ≈ 114.6.
- We **cannot** say: that this is a FOLFOX “feature” or a causal mechanism.
